Mastodon/Scene/Thread/ThreadViewModel.swift

```diff
@@ -55,40 +58,25 @@ class ThreadViewModel {
         self.root = optionalRoot
         self.mastodonStatusThreadViewModel = MastodonStatusThreadViewModel(context: context)
         // end init
-        
-        ManagedObjectObserver.observe(context: context.managedObjectContext)
-            .sink(receiveCompletion: { completion in
-                // do nohting
-            }, receiveValue: { [weak self] changes in
-                guard let self = self else { return }
-                
-                let objectIDs: [NSManagedObjectID] = changes.changeTypes.compactMap { changeType in
-                    guard case let .delete(object) = changeType else { return nil }
-                    return object.objectID
-                }
-                
-                self.delete(objectIDs: objectIDs)
-            })
-            .store(in: &disposeBag)
-        
+
         $root
             .receive(on: DispatchQueue.main)
             .sink { [weak self] root in
                 guard let self = self else { return }
                 guard case let .root(threadContext) = root else { return }
-                guard let status = threadContext.status.object(in: self.context.managedObjectContext) else { return }
+                let status = threadContext.status
                 
                 // bind threadContext
                 self.threadContext = .init(
-                    domain: status.domain,
+                    domain: authContext.mastodonAuthenticationBox.domain, //status.domain,
```

Comment:
I was curious why we need the domain here and it seems like we don't? What do you think?

Reply:
You're right, this was previously used when fetching the cached (domain scoped) statuses from Core Data. I'm removing this.